packages/blade/src/components/LiveAnnouncer/LiveAnnouncer.native.tsx

```diff
@@ -0,0 +1,20 @@
+/* eslint-disable @typescript-eslint/ban-ts-comment */
+
+import { AccessibilityInfo } from 'react-native';
+import type { Assertiveness } from './LiveAnnouncer.d';
+
+export function announce(
+  message: string,
+  _assertiveness: Assertiveness = 'assertive',
+  _timeout = 0,
+): void {
+  AccessibilityInfo.announceForAccessibility(message);
+}
+
+export function clearAnnouncer(_assertiveness: Assertiveness): void {
+  console.log('[Blade LiveAnnouncer]: clearAnnouncer is not needed in react-native');
```

Comment:
We should silently ignore. Consumer should not care about the platform specific errors & implementation details, they should be aware that "okay this does nothing in react-native side". 

Otherwise, say we are using `clearAnnouncer()` in our components, in web it will work but in native will fail (fail the whole component) which is not desirable.

Reply:
Discussed we'll do warning here